src/bindings/html/dom.js

```diff
@@ -77,19 +76,31 @@ export function translateMutations(view, langs, mutations) {
 
   const elements = [];
 
-  targets.forEach(target => target.childElementCount ?
-      elements.push(...getTranslatables(target)) : elements.push(target));
+  targets.forEach(target => {
+    if (target.childElementCount) {
+      elements.push(...getTranslatables(target));
+    } else {
+      if (target.hasAttribute('data-l10n-id')) {
```

Comment:
You can collapse this into a `else if`.  I think I preferred the ternary version but that's just style.